File: Coffee_Shop.java
import java.util.*;

public class Coffee_Shop {
    public static void main(String[] args) {
        System.out.println(" \n     ***Welcome to Saturn Coffee***");

        int price_BlackCoffee = 2;
        int price_Espresso = 3;
        int price_Latte = 4;
        int price_Cappuccino = 5;
        int price_Frappuccino = 6;

        System.out.println("\n1) Black Coffee: $" + price_BlackCoffee);
        System.out.println("2) Espresso: $" + price_Espresso);
        System.out.println("3) Latte: $" + price_Latte);
        System.out.println("4) Cappuccino: $" + price_Cappuccino);
        System.out.println("5) Frapuccino: $" + price_Frappuccino);

        Scanner obj = new Scanner(System.in);
        String order;

        while (true) {
            System.out.print("\nEnter option:");
            order = obj.nextLine();

            if (order.equals("0")) {
                System.exit(0);
            }
            if (order.isEmpty()) {
                System.out.println("I can't read your mind!");
            } else {
                try {
                    int a = Integer.parseInt(order);
                    if (a > 5 || a < 0) {
                        throw new IllegalArgumentException();
                    } else {
                        break;
                    }
                } catch (IllegalArgumentException e) {
                    System.out.println("INVALID INPUT!!");
                }
            }
        }

        String item = "";
        switch (order) {
            case "1" -> item = "Black Coffee";
            case "2" -> item = "Espresso";
            case "3" -> item = "Latte";
            case "4" -> item = "Cappuccino";
            case "5" -> item = "Frappuccino";
            default -> System.out.println("Invalid input!");
        }

        int quantity;
        int confirm;
        System.out.println("\nYou have selected " + item + ".");
        System.out.print("\nHow many " + item + " do you want?");
        quantity = obj.nextInt();

        int total;

        switch (item) {
            case "Black Coffee" -> total = quantity * price_BlackCoffee;
            case "Espresso" -> total = quantity * price_Espresso;
            case "Latte" -> total = quantity * price_Latte;
            case "Cappuccino" -> total = quantity * price_Cappuccino;
            case "Frappuccino" -> total = quantity * price_Frappuccino;
            default -> throw new AssertionError();
        }

        System.out.println("\n     ***Please confirm your order***\n");
        System.out.print(quantity + " " + item + "\n\nYes[1]/No[0]: ");
        confirm = obj.nextInt();

        if (confirm == 1) {
            System.out.print("\nYour total is : $" + total);
        } else if (confirm == 0) {
            System.exit(0);
        }

        System.err.println("");
        System.out.println("\n     ***Payment options***");
        System.out.println("\n1) Cash");
        System.out.println("2) Scan QR code: ");

        String payments;
        obj.nextLine(); // Add this line to consume the newline character
        System.out.print("\nEnter option: ");
        payments = obj.nextLine();

        while (true) {
            if (payments.equals("1")) {
                System.out.print("\nPlease pay at the cash counter and receive your order.\n");
                break;
            }

            if (payments.equals("2")) {
                System.out.println("\n***Payment Successfull***");
                break;
            }

            if (payments.isEmpty()) {
                System.out.println("I can't read your mind!");
            }

            try {
                int a = Integer.parseInt(payments);

                if (a > 2 || a < 0) {
                    throw new IllegalArgumentException();
                } else {
                    break;
                }
            } catch (IllegalArgumentException e) {
                System.out.println("INVALID INPUT");
            }
        }
    }
}

File: Coffee_Shop.py
print("\n*****Welcome to Saturn Coffee*****\n")

price_BlackCoffee=3
price_Espresso=4
price_Latte=6
price_Cappuccino=7
price_Frappuccino=9

print("What would you like to have? Here is our menu:\n")
print("1)Black Coffee: $" + str(price_BlackCoffee))
print("2)Espresso: $" + str(price_Espresso))
print("3)Latte: $" + str(price_Latte))
print("4)Cappuccino: $" + str(price_Cappuccino))
print("5)Frappuccino: $" + str(price_Frappuccino))


id_Black_Coffee = "1"
id_Espresso = "2"
id_Latte = "3"
id_Cappuccino = "4"
id_Frappuccino = "5"

while True:
    order = input("\nEnter option(or 0 to exit): ")
    if order=="0":
       exit()
    
    if not order:
        print("\nI can't read your mind.")
        continue

    
    if order not in [id_Black_Coffee, id_Espresso, id_Latte, id_Cappuccino, id_Frappuccino]:
        print("\nInvalid input!")
        print("Available options are in the menu.")
    else:
  
          if order == id_Black_Coffee:
           item= "Black Coffee"

          elif order == id_Espresso:
           item= "Espresso"

          elif order == id_Latte:
            item ="Latte"

          elif order == id_Cappuccino:
            item="Cappuccino"
            

          elif order == id_Frappuccino:
            item="Frappuccino"  
           
          print("\nSo, you would like to have " + item + ".")
          break
    
while True:
   
   quantity=input("\nHow many " + item + "'s do you want? ")


   if not quantity:
      print("I can't read your mind!")
      continue
   if not quantity.isnumeric():
      print("Invalid input!")
      continue
   
   if quantity=="0":
      exit()
   else:
      break
   
total = 0

if item == "Black Coffee":
    total = price_BlackCoffee * int(quantity)
elif item == "Espresso":
    total = price_Espresso * int(quantity)
elif item == "Latte":
    total = price_Latte * int(quantity)
elif item == "Cappuccino":
    total = price_Cappuccino * int(quantity)
elif item == "Frappuccino":
    total = price_Frappuccino * int(quantity)


print("\nYour total is $" + str(total)+".")

payment_1="1"
payment_2="2"

print("\n        ***Payment options***")
print("1)Cash")
print("2)Scan QR code")

while True:
   pay =input("\nEnter option(or 0 to exit):")

   if not pay.isnumeric():
      print("Invalid input!")
      continue

   if pay=="0":
      exit()

   else:
      break


if pay==payment_1:
      
      input("\nThank You, Here is the order bill.\nPlease pay to the cash counter and receive your order.")
      print("")
      print("Thank you! Visit again!\n")
      
if pay==payment_2:
        
        input("\nThank You! Please scan the QR code.\n")
        print("***Payment Successfull***\n\nHere is your receipt.\nPlease receive your order at the counter.")
        print("\nThank you! Visit again!\n")